common/test/src/IO/CompilationConfigParserTest.cpp

```diff
@@ -248,6 +248,29 @@ TEST_CASE(
   CHECK_THROWS_AS(parser.parse(), ParserException);
 }
 
+TEST_CASE(
+  "CompilationConfigParserTest.parseOneProfileWithNameAndOneDeleteTaskWithMissingTarget",
+  "[CompilationConfigParserTest]")
+{
+  const std::string config(
+    "{\n"
```

Comment:
I prefer to use raw string literals now: `R("...")` as they make the test much easier to read.